app/uploaders/video_cover_uploader.rb

```diff
@@ -0,0 +1,52 @@
+# encoding: utf-8
+
+class VideoCoverUploader < CarrierWave::Uploader::Base
+
+  # Include RMagick or MiniMagick support:
+  # include CarrierWave::RMagick
+  include CarrierWave::MiniMagick
+
+  # Choose what kind of storage to use for this uploader:
+  storage :file
```

Comment:
Why is the storage file here? its going to override the settings from initializer in production and staging as far as I know.